charts/kured/templates/daemonset.yaml

```diff
@@ -97,6 +97,9 @@ spec:
           {{- if .Values.configuration.timeZone }}
             - --time-zone={{ .Values.configuration.timeZone }}
           {{- end }}
+          {{- if .Values.configuration.annotateNode }}
```

Comment:
@evrardjp do we want to remove all changes under charts/** and queue those up for a follow-up PR after this change is reviewed/changed/approved?

Reply:
yes

Reply:
Thanks for the clarification, I've removed those changes.

I have a branch I'm maintaining (off of these changes) that includes the chart updates as a discrete changeset. I'll submit a new PR w/ those changes after this is reviewed/approved/merged.